docker-compose.yml

```
@@ -1,5 +1,14 @@
-version: "3.8"
 services:
+  api-proxy:
+    image: nginx:1.27.3-alpine
```

While it's best practice to use specific versions, since this isn't production supported prefer using a more frequently updated tag so we reduce our maintenance burden.

```suggestion
    image: nginx:1-alpine
```